packages/gatsby-plugin-sharp/src/__tests__/__snapshots__/index.js.snap

```diff
@@ -11,6 +11,25 @@ Object {
 `;
 
 exports[`gatsby-plugin-sharp duotone fixed 1`] = `
+Object {
+  "aspectRatio": 1,
+  "base64": undefined,
+  "density": 72,
+  "originalImg": "/static/1234/7e516/test.png",
+  "originalName": "test.png",
+  "presentationHeight": 100,
+  "presentationWidth": 100,
+  "sizes": "(max-width: 100px) 100vw, 100px",
+  "src": "/static/1234/7e516/test.png",
+  "srcSet": "/static/1234/7991c/test.png 25w,
+/static/1234/a1812/test.png 50w,
+/static/1234/7e516/test.png 100w",
+  "srcSetType": "image/png",
+  "tracedSVG": "data:image/svg+xml,%3csvg%20xmlns='http://www.w3.org/2000/svg'%20width='100'%20height='100'%20viewBox='0%200%20100%20100'%20preserveAspectRatio='none'%3e%3cpath%20d='M41%2024c-18%207-24%2029-11%2043%2015%2017%2044%208%2046-15%201-19-17-34-35-28'%20fill='red'%20fill-rule='evenodd'/%3e%3c/svg%3e",
+}
+`;
+
+exports[`gatsby-plugin-sharp duotone fixed 2`] = `
```

Comment:
Please revert this. This is just a flaky test, not related to prettier. We shouldn't include this specific change.